From: Kill Your TV <killyourtv@i2pmail.org>
Date: Sun, 5 Oct 2014 19:47:18 +0000
Subject: harden

---
 projectfiles/QtCreator/TOX-Qt-GUI.pro | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/projectfiles/QtCreator/TOX-Qt-GUI.pro b/projectfiles/QtCreator/TOX-Qt-GUI.pro
index c2a0547..822f310 100644
--- a/projectfiles/QtCreator/TOX-Qt-GUI.pro
+++ b/projectfiles/QtCreator/TOX-Qt-GUI.pro
@@ -4,6 +4,11 @@
 #
 #-------------------------------------------------
 
+QMAKE_CPPFLAGS *= $(shell dpkg-buildflags --get CPPFLAGS)
+QMAKE_CFLAGS   *= $(shell dpkg-buildflags --get CFLAGS)
+QMAKE_CXXFLAGS *= $(shell dpkg-buildflags --get CXXFLAGS)
+QMAKE_LFLAGS   *= $(shell dpkg-buildflags --get LDFLAGS)
+
 defineTest(minQtVersion) {
     maj = $$1
     min = $$2
